rest-api/alembic/versions/574efa4de1ba_ppi_drop_withdrawals.py

```diff
@@ -222,27 +213,84 @@ def downgrade(engine_name):
     INNER JOIN participant_summary ps ON p.participant_id = ps.participant_id
     LEFT OUTER JOIN hpo ON p.hpo_id = hpo.hpo_id
     LEFT OUTER JOIN code ac ON qra.value_code_id = ac.code_id
-    WHERE (ps.email IS NULL OR ps.email NOT LIKE '%@example.com') AND
-           (hpo.name IS NULL OR hpo.name != 'TEST') AND
-           /* Blacklist any codes which are superfluous for PPI QA. */
-           qc.value NOT IN (
-             'ExtraConsent_DataSharingVideo',
-             'ExtraConsent_EmailACopyToMe',
-             'ExtraConsent_FitnessTrackerVideo',
-             'ExtraConsent_HealthDataVideo',
-             'ExtraConsent_HealthRecordVideo',
-             'ExtraConsent_KeepinginTouchVideo',
-             'ExtraConsent_OtherHealthDataVideo',
-             'ExtraConsent_PhysicalMeausrementsVideo', # Intentional typo
-             'ExtraConsent_WelcomeVideo',
-             'YoutubeVideos_WhatAreWeAsking',
-             'YoutubeVideos_RiskToPrivacy'
-           )
+    WHERE p.withdrawal_status = 1 AND # NOT_WITHDRAWN
+        (ps.email IS NULL OR ps.email NOT LIKE '%@example.com') AND
+        (hpo.name IS NULL OR hpo.name != 'TEST') AND
+        /* Blacklist any codes which are superfluous for PPI QA. */
+        qc.value NOT IN (
+          'ExtraConsent_DataSharingVideo',
+          'ExtraConsent_EmailACopyToMe',
+          'ExtraConsent_FitnessTrackerVideo',
+          'ExtraConsent_HealthDataVideo',
+          'ExtraConsent_HealthRecordVideo',
+          'ExtraConsent_KeepinginTouchVideo',
+          'ExtraConsent_OtherHealthDataVideo',
+          'ExtraConsent_PhysicalMeausrementsVideo', # Intentional typo
+          'ExtraConsent_WelcomeVideo',
+          'YoutubeVideos_WhatAreWeAsking',
+          'YoutubeVideos_RiskToPrivacy'
+        )
+"""
+
+
+_MEASUREMENTS_VIEW_SQL = """
+CREATE OR REPLACE VIEW physical_measurements_view AS
+ SELECT
+   p.participant_id participant_id,
+   YEAR(p.sign_up_time) participant_sign_up_year,
+   p.withdrawal_status participant_withdrawal_status,
```

Comment:
You can remove this and the next line now,  and also from questionnaire_response_answer_view.

Reply:
Done, thanks.